exasol/toolbox/nox/_test.py

```diff
@@ -21,14 +21,13 @@
 def _test_command(
     path: Path, config: Config, context: MutableMapping[str, Any]
 ) -> Iterable[str]:
-    base_command = ["poetry", "run"]
     coverage_command = (
         ["coverage", "run", "-a", f"--rcfile={config.root / 'pyproject.toml'}", "-m"]
         if context["coverage"]
         else []
     )
     pytest_command = ["pytest", "-v", f"{path}"]
-    return base_command + coverage_command + pytest_command + context["fwd-args"]
+    return coverage_command + pytest_command + context["fwd-args"]
```

Comment:
adapted this to work for cases like: https://github.com/exasol/github-issue-adapter/blob/refactoring/%23422-Replace_pytest_plugin_itde_by_backend/.github/workflows/ci.yml#L101

tested by checking out https://github.com/exasol/github-issue-adapter/pull/427/files, copying of the functions associated with `test:coverage` to the `noxfile.py` and running `poetry run -- nox -s test:coverage -- --backend=onprem`, which ran both the unit & integration tests.

Reply:
:+1: